apiary.apib

```diff
@@ -9690,15 +9705,16 @@ Fetch which version of Projects the account is using.
         + product_id: `d905ff57-e866-0f59-9d1e-1fd4538bfae1` (string, optional)
 
 ## NoteSubjectTypes (enum)
-+ `contact`
 + `company`
++ `contact`
++ `creditNote`
 + `deal`
 + `invoice`
-+ `creditNote`
-+ `subscription`
++ `nextgenProject`
 + `product`
++ `project`
```

Comment:
[notes.create](https://github.com/teamleadercrm/api/blob/master/src/01-general/notes.apib#L35) does not allow passing a project though.
These types should be split up since it doesn't make sense to still add support old projects in this endpoint